framework/test/unit/modules/interoperability/base_interoperability_module.spec.ts

```diff
@@ -0,0 +1,730 @@
+import { MAX_UINT64 } from '@liskhq/lisk-validator';
+import {
+	activeValidator,
+	activeValidators,
+	chainData,
+	chainInfo,
+	chainValidators,
+	channelData,
+	contextWithValidValidatorsHash,
+	createInitGenesisStateContext,
+	genesisInteroperability,
+	getStoreMock,
+	lastCertificate,
+	terminatedOutboxAccount,
+	terminatedStateAccount,
+} from './interopFixtures';
+import {
+	ActiveValidator,
+	ChainStatus,
+	EMPTY_BYTES,
+	MainchainInteroperabilityModule,
+} from '../../../../src';
+import {
+	MAX_NUM_VALIDATORS,
+	MIN_RETURN_FEE_PER_BYTE_BEDDOWS,
+} from '../../../../src/modules/interoperability/constants';
+import { CreateGenesisBlockContextParams, InMemoryPrefixedStateDB } from '../../../../src/testing';
+import { PrefixedStateReadWriter } from '../../../../src/state_machine/prefixed_state_read_writer';
+import { computeValidatorsHash } from '../../../../src/modules/interoperability/utils';
+import { OwnChainAccountStore } from '../../../../src/modules/interoperability/stores/own_chain_account';
+import { ChainAccountStore } from '../../../../src/modules/interoperability/stores/chain_account';
+import { ChannelDataStore } from '../../../../src/modules/interoperability/stores/channel_data';
+import { ChainValidatorsStore } from '../../../../src/modules/interoperability/stores/chain_validators';
+import { OutboxRootStore } from '../../../../src/modules/interoperability/stores/outbox_root';
+import { TerminatedStateStore } from '../../../../src/modules/interoperability/stores/terminated_state';
+import { TerminatedOutboxStore } from '../../../../src/modules/interoperability/stores/terminated_outbox';
+
+describe('initGenesisState Common Tests', () => {
```

Comment:
you can move `_verifyChainID` related tests here, we can have describe blocks based on the method names and related tests within that for better maintainability